src/overlays/actors/ovl_En_Fr/z_en_fr.c

```diff
@@ -27,99 +183,1134 @@ const ActorInit En_Fr_InitVars = {
     (ActorFunc)EnFr_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Fr/func_80A1A9A0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Fr/EnFr_Init.s")
+static EnFrColor sEnFrColor[5] = { { 200, 170, 0, 255 },   // FROG_YELLOW
+                                   { 0, 170, 200, 255 },   // FROG_BLUE
+                                   { 210, 120, 100, 255 },   // FROG_RED
+                                   { 120, 130, 230, 255 },   // FROG_PURPLE
+                                   { 190, 190, 190, 255 } }; // FROG_WHITE
+
+// Jumping back into water frog animation
+//      sLogSpotToFromWater[frog].xzDist is magnitude of xz distance frog travels
+//      sLogSpotToFromWater[frog].yaw is rot around y-axis of jumping back into water
+//      sLogSpotToFromWater[frog].yDist is change in y distance frog travels
+static LogSpotToFromWater sLogSpotToFromWater[6] = {
+    { 0.0, 0.0, 0.0 },              // Prop   (Where link pulls ocarina)
```

Comment:
make sure to put f on the end of your floats

Reply:
additionally, we try not to specify array lengths when possible. if people want to modify an array at a later time it would avoid throwing errors